Sources/TuistCoreTesting/Models/Target+TestData.swift

```diff
@@ -11,8 +11,8 @@ public extension Target {
                      productName: String? = nil,
                      bundleId: String = "com.test.bundle_id",
                      deploymentTarget: DeploymentTarget? = .iOS("13.1", [.iphone, .ipad]),
-                     infoPlist: InfoPlist? = .file(path: AbsolutePath("/Info.plist")),
```

Comment:
I changed this because it was crashing integration tests as it they were expecting to find a file at ```/Info.plist"``` and I couldn't write a file at "/`.
Also I think it's better to assume `testData` are initialized with empty/nil values, to not have surprises. Better to be explicit when you want something optional to not be nil